swag_client/schemas/v2.py

```diff
@@ -46,10 +46,10 @@ class RegionSchema(Schema):
 
 
 class AccountSchema(Schema):
-    schemaVersion = fields.Str(missing='2')
+    schemaVersion = fields.Str(missing='v2')
```

Comment:
should this be `v2` or just `2`? Let's leave the default.

Reply:
went back to the default. reason i had changed this is because it's non-standard. We use 'v1' as the default for v1 schema and '2' as the default for v2. Also  people can provide either v2 or 2